Taxi/Taxi/Presenter/CellView.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  CellView.swift
+//  Taxi
+//
+//  Created by Yosep on 2022/06/09.
+//
+import SwiftUI
+
+struct CellView: View {
+    let party: TaxiParty
+    private let paddingSize: CGFloat = 6
```

Comment:
body와 data property들 간에 개행이 하나 있으면 구분하기 쉬울 것 같아요 !
```suggestion
    private let paddingSize: CGFloat = 6

```

Reply:
네, 다른부분들도 반영 완료하였습니다.